plugins/attachment-resources/src/components/AttachmentStyleBoxEditor.svelte

```diff
@@ -64,6 +64,7 @@
     const old = getAttribute(client, object, key)
     if (description !== old) {
       await updateAttribute(client, object, object._class, key, description)
+      haveUnsavedChanges = false
```

Comment:
Should we move this out of the if statement? I believe after `save` we can consider that there are no unsaved changes.

Reply:
You are right we should do this, thanks